content/docs/languages/go/basics.md

```diff
@@ -486,24 +492,21 @@ r := rand.New(rand.NewSource(time.Now().UnixNano()))
 pointCount := int(r.Int31n(100)) + 2 // Traverse at least two points
 var points []*pb.Point
 for i := 0; i < pointCount; i++ {
-	points = append(points, randomPoint(r))
+  points = append(points, randomPoint(r))
 }
 log.Printf("Traversing %d points.", len(points))
 stream, err := client.RecordRoute(context.Background())
 if err != nil {
-	log.Fatalf("%v.RecordRoute(_) = _, %v", client, err)
+  log.Fatalf("%v.RecordRoute(_) = _, %v", client, err)
 }
 for _, point := range points {
-	if err := stream.Send(point); err != nil {
-		if err == io.EOF {
-			break
-		}
```

Comment:
Note that the example code in the repo doesn't contain this test for `io.EOF`, which is why it is removed. (I'd expect to see such a test for a `Recv`, not a send. Anyhow, let me know if this is ok.)

Reply:
We should really get rid of all the `log.Fatal`s in example server handlers.  Errors happen while sending and receiving all the time due to connection loss and that shouldn't take down your server.

Filed https://github.com/grpc/grpc-go/issues/3952 to fix.